File: competitive-programming/cpbook/Chapter_01/1-es/3-uva-11614.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Integer t = scanner.nextInt();
        Double n, a, b, c, k, delta;

        while (t-- != 0) {
            n = scanner.nextDouble();
            a = 1.0;
            b = 1.0;
            c = -2.0 * n;

            delta = Math.pow(b, 2.0) - (4.0 * a * c);
            k = (((-b) + Math.sqrt(delta))) / (2.0 * a);

            System.out.println(k.intValue());
        }

        scanner.close();
    }
}

File: competitive-programming/cpbook/Chapter_01/1-es/3-uva-11614.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int t;
    cin >> t;

    double n, a, b, c, delta;
    while (t--)
    {
        cin >> n;

        a = 1.0;
        b = 1.0;
        c = -2.0 * n;
        delta = pow(b, 2) - (4.0 * a * c);

        long long k = (long long)(((-b + sqrt(delta)) / (2.0 * a)));

        cout << k << endl;
    }

    return 0;
}